src/pytorch/__init__.py

```diff
@@ -1,5 +1,10 @@
 '''
 High-performance PyTorch operations for neural network potentials
 '''
+import os.path
+import site
+import torch
+torch.ops.load_library(os.path.join(site.getsitepackages()[-1],"NNPOps", "libNNPOpsPyTorch.so"))
```

Comment:
`site.getsitepackages` returns a list of paths. I think, we check all of them for `NNPOps/libNNPOpsPyTorch.so` and load the first found.